src/content/getusermedia/resolution/js/main.js

```diff
@@ -70,7 +70,9 @@ video.onloadedmetadata = displayVideoDimensions;
 
 function getMedia(constraints) {
   if (stream) {
-    stream.getTracks().forEach(function(track) { track.stop(); });
+    stream.getTracks().forEach(function(track) {
```

Comment:
Yes — though we can turn this off. 

Also disallowed because of space after curly brace (likewise, can turn this off).